source/appModules/azardi-2_0.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: UTF-8 -*-
+#appModules/azardi-2_0.py
+#A part of NonVisual Desktop Access (NVDA)
+#Copyright (C) 2016 Noelia Ruiz Martínez
+#This file is covered by the GNU General Public License.
+#See the file COPYING for more details.
+
+import appModuleHandler
+import controlTypes
+import winUser
+import api
+from NVDAObjects.IAccessible.mozilla import Document
+from NVDAObjects.IAccessible.sysTreeView32 import TreeViewItem
+
+class AzardiDocument(Document):
+
+	role = controlTypes.ROLE_DOCUMENT
+
+class AzardiTreeViewItem(TreeViewItem):
+
+	def script_enter(self, gesture):
+		api.moveMouseToNVDAObject(self)
+		api.setMouseObject(self)
+		winUser.mouse_event(winUser.MOUSEEVENTF_LEFTDOWN,0,0,None,None)
```

Comment:
For these scripts, it would be really useful if we had a comment to describe why this needs to be done.